index.js

```diff
@@ -0,0 +1,66 @@
+class TinyPagination {
+
+    pageNo = 1
+
+    constructor(pageSize, data) {
+        this.pageSize = pageSize
+        this.data = data
+    }
+
+    getPageData(pageNo) {
+        this.pageNo = pageNo
+        const offset = (pageNo - 1) * this.pageSize
+        return (offset + this.pageSize >= this.data.length) ? this.data.slice(offset, this.data.length) : this.data.slice(offset, offset + this.pageSize);
+    }
+
+    getPageLength() {
+        return Math.ceil(this.data.length / this.pageSize)
+    }
+
+    getCurrentPageNo() {
+        return this.pageNo
+    }
+
+    setData(newData) {
+        this.data = newData
+    }
+
+    nextPage() {
+        return this.getPageData(this.pageNo + 1)
+    }
+
+    previousPage() {
+        return this.getPageData(this.pageNo - 1)
+    }
```

Comment:
It would be better to add a validation to check if `pageNo` isn't larger than the pages array length that way it will stop when the pages are done.

Reply:
Yeah, it's good advice and I'v thought about it, but I didn’t have time to finishe it. Lol. 
Thank you very much!